src/rprblender/nodes/__init__.py

```diff
@@ -47,11 +50,12 @@ def poll(cls, context):
         NodeItem('ShaderNodeVolumeInfo'),
         NodeItem('RPRShaderProceduralUVNode'),
         NodeItem('RPRShaderNodeLookup'),
-    ],),
-    RPR_ShaderNodeCategory('RPR_OUTPUT', "Output", items=[
+        NodeItem('ShaderNodeLightFalloff'),
```

Comment:
We don't support this node.  Why add it?

Reply:
Yes, you are right. I'll fix it.